pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -282,6 +282,10 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 		if cr.Spec.Sharding.Mongos.Expose.ExposeType == "" {
 			cr.Spec.Sharding.Mongos.Expose.ExposeType = corev1.ServiceTypeClusterIP
 		}
+
+		if len(cr.Spec.Sharding.Mongos.ServiceAccountName) == 0 {
```

Comment:
Maybe we need to have it only for >= 1.16.0

Reply:
yes, strange why I skipped it. ((